spec/virtual_total_spec.rb

```diff
@@ -435,6 +435,7 @@ def model_with_children(count)
       let(:author3) { Author.create }
       let(:author4) { Author.create.tap { |a| a.create_books(1, :published => true) } }
 
+      describe ":sum" do
       # NOTE: rails converts the nil to a 0
       it "calculates sum with one off query" do
```

Comment:
probably want to indent these

Reply:
think you're looking at an old diff, sorry, these are totally indented currently